vulcanai2/tests/models/test_layers.py

```diff
@@ -78,8 +78,8 @@ def test_create_denseunit(self, test_denseunit_parameters):
         """Check if passing wrong parameters raises TypeError."""
         with pytest.raises(TypeError) as e:
             DenseUnit(**test_denseunit_parameters)
-            assert e.message == "__init__() got an unexpected \
-                                keyword argument 'in_channels'"
+        assert e.message == "__init__() got an unexpected \
```

Comment:
I get the following error ```assert e.message == "__init__() got an unexpected \
                                keyword argument 'in_channels'"
E       AttributeError: 'ExceptionInfo' object has no attribute 'message'```.

Reply:
Hmm I just tested now, I'm getting the same. Not sure why the error message changed, but this could be because from the merge commits I guess

Reply:
>~but this could be because from the merge commits I guess~

Its actually cuz I moved the assert out of `with`